src/shutdown.py

```diff
@@ -8,52 +8,67 @@
 from helper_threading import StoppableThread
 from knownnodes import saveKnownNodes
```

Comment:
Place imports in a order

Reply:
importing shared and state in wrong order could trigger bugs.